gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,399 @@
+					   WHERE attname = 'tablespace_location' AND
+					         NOT attisdropped AND
+					         attrelid = 'arenadata_toolkit.__db_files_current'::regclass)
+		THEN
+			CREATE OR REPLACE VIEW arenadata_toolkit.__db_files_current
+			AS
+			SELECT
+				c.oid AS oid,
+				c.relname AS table_name,
+				n.nspname AS table_schema,
+				c.relkind AS type,
+				c.relstorage AS storage,
+				d.datname AS table_database,
+				t.spcname AS table_tablespace,
+				dbf.segindex AS content,
+				dbf.segment_preferred_role AS segment_preferred_role,
+				dbf.hostname AS hostname,
+				dbf.address AS address,
+				dbf.full_path AS file,
+				dbf.size AS file_size,
+				dbf.modified_dttm AS modifiedtime,
+				dbf.changed_dttm AS changedtime,
+				CASE
+					WHEN 'pg_default' = t.spcname THEN gpconf.datadir || '/base'
+					WHEN 'pg_global' = t.spcname THEN gpconf.datadir || '/global'
+					ELSE (SELECT tblspc_loc
+						  FROM gp_tablespace_segment_location(t.oid)
+						  WHERE gp_segment_id = dbf.segindex)
+					END AS tablespace_location
+			FROM arenadata_toolkit.__db_segment_files dbf
+			LEFT JOIN pg_class c ON c.oid = dbf.reloid
+			LEFT JOIN pg_namespace n ON c.relnamespace = n.oid
+			LEFT JOIN pg_tablespace t ON dbf.tablespace_oid = t.oid
+			LEFT JOIN pg_database d ON dbf.datoid = d.oid
+			LEFT JOIN gp_segment_configuration gpconf ON dbf.dbid = gpconf.dbid;
+		END IF;
+	END IF;
+END $$;
+
+DO $$
+BEGIN
+	IF EXISTS (SELECT 1
+			   FROM pg_views
+			   WHERE schemaname = 'arenadata_toolkit' AND
+			         viewname = '__db_files_current_unmapped')
+	THEN
+		-- Use "IF" into another "IF" because in other case "<table_name>::regclass"
+		-- leads to error if the such table does not exist.
+		IF NOT EXISTS (SELECT 1
+					   FROM pg_attribute
+					   WHERE attname = 'tablespace_location' AND
+					         NOT attisdropped AND
+					         attrelid = 'arenadata_toolkit.__db_files_current_unmapped'::regclass)
+		THEN
+			CREATE OR REPLACE VIEW arenadata_toolkit.__db_files_current_unmapped
+			AS
+			SELECT
+				v.table_database,
+				v.table_tablespace,
+				v.content,
+				v.segment_preferred_role,
+				v.hostname,
+				v.address,
+				v.file,
+				v.file_size,
+				v.tablespace_location
+			FROM arenadata_toolkit.__db_files_current v
+			WHERE v.oid IS NULL;
+		END IF;
+	END IF;
+END $$;
+
+DO $$
+BEGIN
+	IF EXISTS (SELECT 1
+			   FROM pg_tables
+			   WHERE schemaname = 'arenadata_toolkit' AND
+			         tablename = 'db_files_history')
+	THEN
+		-- Use "IF" into another "IF" because in other case "<table_name>::regclass"
+		-- leads to error if the such table does not exist.
+		IF NOT EXISTS (SELECT 1
+					   FROM pg_attribute
+					   WHERE attname = 'tablespace_location' AND
+					         NOT attisdropped AND
+					         attrelid = 'arenadata_toolkit.db_files_history'::regclass)
+		THEN
+			EXECUTE FORMAT($fmt$ALTER TABLE arenadata_toolkit.db_files_history
+								RENAME TO db_files_history_backup_%1$s;$fmt$,
```

Comment:
```suggestion
								RENAME TO arenadata_toolkit.%1$I;$fmt$,
				to_char(now(), '"db_files_history_backup_"YYYYMMDD"T"HH24MISS'));
```

Reply:
1) specification contains upper `T` between date and time
2) qualificator `I` corresponds to identifier

Reply:
> alter table arenadata_toolkit.db_files_history rename to db_files_history_backup_20231221T141316

try
```sql
alter table arenadata_toolkit.db_files_history rename to "db_files_history_backup_20231221T141316";
```
or use `I` qualificator

Reply:
Done.